src/tests/Reader/NRFReader.t.h

```diff
@@ -0,0 +1,55 @@
+#include <cxxtest/TestSuite.h>
+
+#include <SourceTerm/NRFReader.h>
+#include <SourceTerm/NRF.h>
+
+#include "slipRatesData.h"
+
+namespace seissol {
+  namespace unit_test {
+    class NRFReaderTestSuite;
+  }
+}
+
+class seissol::unit_test::NRFReaderTestSuite : public CxxTest::TestSuite
+{
+public:
+  void testNRFReader()
+  {
+    //NRF stores all values always in double, so use eps = 1e-16
+    constexpr double eps = 1e-16;
```

Comment:
Is there a reason for the eps instead of numerical eps?
The comment doesn't really explain the why